jdi-light-mobile-html-web-tests/README.md

```diff
@@ -0,0 +1,112 @@
+#How to launch HTML Web tests on Android emulator or iOS simulator
+### Requirements for Android
+Required tools for Android: Android Studio, Appium, Maven.<br>
+
+1. Download **chrome driver** For more details
```

Comment:
The same issue for numbering paragraphs all are 1

Reply:
As it's markdown format, all items should be 1. - it's correct
https://guides.github.com/features/mastering-markdown/